cmd/operator/main.go

```diff
@@ -125,7 +126,7 @@ func init() {
 	flagset.StringVar(&cfg.ThanosDefaultBaseImage, "thanos-default-base-image", "improbable/thanos", "Thanos default base image")
 	flagset.Var(ns, "namespaces", "Namespaces to scope the interaction of the Prometheus Operator and the apiserver.")
 	flagset.Var(&cfg.Labels, "labels", "Labels to be add to all resources created by the operator")
-	flagset.StringVar(&cfg.CrdGroup, "crd-apigroup", monitoringv1.Group, "prometheus CRD  API group name")
+	flagset.StringVar(&cfg.CrdGroup, "crd-apigroup", monitoring.GroupName, "prometheus CRD  API group name")
```

Comment:
for what it's worth, if this makes any problems this might as well be removed, it was introduced as an experimental flag that may be removed at any point

Reply:
but it seems that it didn't cause that much complications, so let's keep it as is

Reply:
:+1: I stumbled across and was having raised eye-brows if this actually works, when set to something else. I didn't touch it for backwards compatibility.